ci/unreal-engine.version

```diff
@@ -1 +1 @@
-UnrealEngine-cccf25f9db4a83510a53e3d9a84c7faea8b975c8
```

Comment:
This file is parsed by some scripts, I hope the newline is not an issue (as long as CI passes I'm fine with the change).